rskj-core/src/main/java/co/rsk/peg/BridgeStorageProvider.java

```diff
@@ -912,6 +912,13 @@ protected void saveNextPegoutHeight() {
         safeSaveToRepository(NEXT_PEGOUT_HEIGHT_KEY, nextPegoutHeight, BridgeSerializationUtils::serializeLong);
     }
 
+    protected int getReleaseRequestQueueSize() {
+        if (releaseRequestQueue != null && activations.isActive(RSKIP271)) {
```

Comment:
No need to check for activations here. The queue size is not a consensus change

Reply:
Also, you should be calling `getReleaseRequestQueue` method to get the queue from storage if not present in the memory

Reply:
Resolved